arch/risc-v/src/qemu-rv/qemu_rv_start.c

```diff
@@ -60,10 +60,15 @@ uintptr_t g_idle_topstack = QEMU_RV_IDLESTACK_TOP;
  * Name: qemu_rv_start
  ****************************************************************************/
 
-void qemu_rv_start(void)
+void qemu_rv_start(int mhartid)
 {
   uint32_t *dest;
 
+  if (0 < mhartid)
```

Comment:
```suggestion
 if (mhartid > 0)
```
I think for equal comparison if (0 == ...) it is ok, but for greater/lower comparison it doesn't help and make more difficult to read

Reply:
Thanks